core/src/main/java/org/apache/cxf/common/util/ClassHelper.java

```diff
@@ -28,9 +28,15 @@
  *
  */
 public class ClassHelper {
+
+    public static final String USE_DEFAULT_CLASS_HELPER = "org.apache.cxf.useDefaultClassHelper";
+
     static final ClassHelper HELPER;
+    static final ClassHelper CXF_HELPER;
+
     static {
         HELPER = getClassHelper();
```

Comment:
I think more often than not we will create 2 instances of `ClassHelper` (HELPER  and DEFAULT_HELPER would be the same), I would suggest to do that:

```
DEFAULT_HELPER = new ClassHelper();
HELPER = getClassHelper(DEFAULT_HELPER);
```
And

```
private static ClassHelper getClassHelper(ClassHelper defaultHelper) {
 ...
    return defaultHelper;
}
```

So we would be using the same instance in case Spring helpers are not being asked for.